source/_integrations/smarthab.markdown

```diff
@@ -17,8 +18,10 @@ devices and you have access to their app-based services, you will be able
 to control your lights and shutters with the SmartHab integration for Home 
 Assistant.
 
-Once you have added a `smarthab` entry to your configuration, your supported 
-devices will automatically be discovered and made available on your dashboard.
+<div class='note info'>
+The easiest way to set up this integration is to do it from the UI. If you
+prefer to do it from `configuration.yaml`, follow these instructions.
+</div>
```

Comment:
This is not really something for a note. It is better to add instructions on how to add the integration via the UI and then the alternative option below, using yaml.